docs/examples/gnmi-clients/gnmic/index.md

```diff
@@ -61,18 +49,881 @@ gnmic -a 127.0.0.1:6030 -u admin -p admin --insecure capabilities  \
+
+```javascript
+{
+  "timestamp": 1626305876151537555,
+  "time": "2021-07-15T00:37:56.151537555+01:00",
+  "results": [
+    {
+      "operation": "UPDATE",
+      "path": "network-instances/network-instance[name=default]/protocols/protocol[name=BGP]/bgp/neighbors/neighbor[neighbor-address=10.10.100.43]"
+    }
+  ]
+}
+```
+
+#### Configure BGP neighbor address, peer AS and send-community
+
+```shell
+gnmic -a 10.83.13.214:6030 -u cvpadmin -p arastra --insecure --gzip set  \
+--update-path '/network-instances/network-instance[name=default]/protocols/protocol[name=BGP]/bgp/neighbors/neighbor[neighbor-address=10.10.100.43]' \
+--update-file value.json
+```
+
+`value.json`:
+
+`{"config": {"neighbor-address":"10.10.100.43", "peer-as": 123, "enabled": true, "send-community": "EXTENDED"}}`
+
+Output:
+
+```javascript
+{
+  "timestamp": 1626305480401353997,
+  "time": "2021-07-15T00:31:20.401353997+01:00",
+  "results": [
+    {
+      "operation": "UPDATE",
+      "path": "network-instances/network-instance[name=default]/protocols/protocol[name=BGP]/bgp/neighbors/neighbor[neighbor-address=10.10.100.43]"
+    }
+  ]
+}
+```
+
+#### Create BGP peer group
+
+```shell
+gnmic -a 10.83.13.214:6030 -u cvpadmin -p arastra --insecure --gzip set \
+  --update-path '/network-instances/network-instance[name=default]/protocols/protocol[name=BGP]/bgp/peer-groups/peer-group[peer-group-name=XYZ]' \
+  --update-file value.json
+```
+
+`value.json`:
+
+`{"config": {"peer-group-name":"XYZ", "local-as": 114}}`
+
+#### Update BGP peer AS
+
+```shell
+gnmic -a 10.83.13.214:6030 -u cvpadmin -p arastra --insecure --gzip set \
+--update-path '/network-instances/network-instance[name=default]/protocols/protocol[name=BGP]/bgp/neighbors/neighbor[neighbor-address=10.10.100.43]/config/peer-as' \
+--update-value '110'
+```
+
+```javascript
+{
+  "timestamp": 1626306067189329813,
+  "time": "2021-07-15T00:41:07.189329813+01:00",
+  "results": [
+    {
+      "operation": "UPDATE",
+      "path": "network-instances/network-instance[name=default]/protocols/protocol[name=BGP]/bgp/neighbors/neighbor[neighbor-address=10.10.100.43]/config/peer-as"
+    }
+  ]
+}
+```
+
+#### Update BGP peer group
+
+```shell
+gnmic -a 10.83.13.214:6030 -u cvpadmin -p arastra --insecure --gzip set \
+--update-path '/network-instances/network-instance[name=default]/protocols/protocol[name=BGP]/bgp/neighbors/neighbor[neighbor-address=10.10.100.43]/config/peer-group' \
+--update-value 'XYZ'
+```
+
+```javascript
+{
+  "timestamp": 1626306067189329813,
+  "time": "2021-07-15T00:41:07.189329813+01:00",
+  "results": [
+    {
+      "operation": "UPDATE",
+      "path": "network-instances/network-instance[name=default]/protocols/protocol[name=BGP]/bgp/neighbors/neighbor[neighbor-address=10.10.100.43]/config/peer-as"
+    }
+  ]
+}
+```
+
+#### Create an ACL
+
+```shell
+
+gnmic -a 10.83.13.108:6030 -u cvpadmin -p arastra --insecure --gzip set \
+--update-path /acl/acl-sets \
+--update-file acl2.json
+{
```

Comment:
we should probably split out the command from the output to make the copy operations discrete for readers.